### DISCUSSION TITLE
[Allure-Gauge-java] There is no test body log after generating report from xml file

### DISCUSSION THREAD

**Discussion by Hungleminh** [allure-framework/allure2#2737] [Questions & Support]
Hi team,

After executing (gauge java maven by "mvn gauge@execute Dpath="my_path""), it generates xml report (using xml-report plugins). And then, i generate allure report by command "allure generate <my_path_xml> --clean. My problem is that i can start viewing the report but there is no test body logged in it, please see the attached screenshot. Can someone advise me in this case?

Note: from xml result file, it only displays testname and there is no more information.

<img width="267" alt="Screenshot 2024-09-20 at 11 10 55" src="https://github.com/user-attachments/assets/e02ff524-7ed8-46e4-b33d-fcae37a8376e">
 
![Screenshot 2024-09-20 at 11 13 27](https://github.com/user-attachments/assets/b695630d-9a71-4c26-b240-3213bff3ff8c)

**Comment by Hungleminh** [comment]
Hi, anyone has an idea on this?

**Reply by baev** [reply]
Allure displays all the information available in an XML report. Please see the related topic https://github.com/getgauge/taiko/discussions/2689 and the issue https://github.com/getgauge/gauge/issues/2438

**Reply by Hungleminh** [reply]
Hi @baev , do you mean that it's not a problem of allure but xml-report plugins?
